src/main/java/net/phptravels/gui/login/HomePage.java

```diff
@@ -17,84 +15,216 @@
     
-    @FindBy(id = "date")
+    @FindBy(xpath = "//form[@id = 'tours-search']//input[@id = 'date']")
     private ExtendedWebElement toursDate;
     
-    @FindBy(xpath = "//a//p[text()= 'Travellers ']")
-    private ExtendedWebElement travellersDropdown;
+    @FindBy(xpath = "//form[@id = 'tours-search']//input[@id = 'tours_adults']")
+    private ExtendedWebElement toursAdultsInput;
     
-    @FindBy(id = "select2-from_country-container")
-    private ExtendedWebElement fromCountryTextbox;
+    @FindBy(xpath = "//form[@id = 'tours-search']//input[@id = 'tours_child']")
+    private ExtendedWebElement toursChildsInput;
     
-    @FindBy(id = "select2-to_country-container")
-    private ExtendedWebElement toCountryTextbox;
+    @FindBy(xpath = "//form[@id = 'tours-search']//a[@class = 'dropdown-toggle dropdown-btn travellers waves-effect']")
+    private ExtendedWebElement toursTravellersDropdown;
     
-    @FindBy(xpath ="//div[text()= '28-04-2022']")
+    @FindBy(xpath = "//form[@id = 'tours-search']//button[@id = 'submit']")
+    private ExtendedWebElement toursSearchButton;
+    
+    @FindBy(xpath = "//div//button[@id = 'visa-tab']")
+    private ExtendedWebElement visaButton;
+    
+    @FindBy(xpath = "//div[@id='visa']//span[@data-select2-id = 5]")
+    private ExtendedWebElement visaFromCountryComboBox;
+    
+    @FindBy(xpath = "//div[@id='visa']//span[@data-select2-id = 7]")
+    private ExtendedWebElement visaToCountryComboBox;
+    
+    @FindBy(xpath ="//div[@id='visa']//input[@id = 'date']")
     private ExtendedWebElement visaDate;
     
+    @FindBy(xpath = "//div[@id='visa']//button[@type = 'submit']")
+    private ExtendedWebElement visaSubmitButton;
+     
+    @FindBy(xpath = "//input[@class='select2-search__field']")
+    private ExtendedWebElement universalInputField;
+    
+    @FindBy(xpath = "//span[@class='select2-results']//li")
+    private ExtendedWebElement universalSearchResult;
+    
     public HomePage(WebDriver driver) {
         super(driver);
     }
 
     public Topbar topbar() {
         return topbar;
     }
+    
+    
+    public void hotelSearchForm(String cityName, String checkin, String checkout, String rooms, String adults, String childs) {
+    	
+    	hotelDestinationCombobox.click();
+    	universalInputField.type(cityName);
+    	universalSearchResult.click();
+    	
+    	hotelCheckinDate.click();
+    	hotelCheckinDate.type(checkin);
+    	hotelCheckoutDate.click();
+    	hotelCheckoutDate.type(checkout);
+    	
+    	hotelTravellersCombobox.click();
+    	hotelRoomsInput.type(rooms);
+    	hotelAdultsInput.type(adults);
+    	hotelCheckinDate.type(childs); 
+    	hotelNationalitySelect.click();
+    	hotelNationalitySelect.select("Argentina");
+    	
+    	hotelSearchButton.click();
+    }
+    
+    public void flightsSearchForm(String flyingFrom, String toDestination, String departureDate, String adults, String childs, String infants) {
```

Comment:
String flyingFrom, String toDestination
Make names of variables more similar
For example flyingFrom and flyingTo